### DISCUSSION TITLE
Should we use Confirmatory Factor Analysis (CFA) as an instrument assessment method?

### DISCUSSION THREAD

**Discussion by kylefox1** [Ledgerback/DGSF#2] [Q&A]
I read [B Impact Assessment as a Sustainable Tool: Analysis of the Certification Model](https://www.mdpi.com/2071-1050/14/9/5590) recently, and it made me want to ask the question in the post's title.

What are your thoughts on CFA and applying it here?

**Comment by linxule** [comment]
![image](https://github.com/Ledgerback/DGSF/assets/43122877/50039327-2a6a-405c-9199-2ba2c0df2dab)
I wonder to what extent do we want to cover the various aspects? 

1. The environment and customers aspects seem to be difficult to capture?  
2. DEI would be hard to assess, especially given recent backlashes in the academic world. Would it be ok if we assess what members are interested in? Like what projects and which groups of projects they have contributed to?

**Reply by linxule** [reply]
Taking a step back, regardless of what we decide, it might be helpful to choose one DAO to evaluate with the framework and try to get some feedback from the DAO community?

**Reply by kylefox1** [reply]
The authors of the [DAO Health Survey](https://talentdao.substack.com/p/nodw-16-developing-the-dao-health) used factor analysis, specifically EFA, for analyzing their instrument to conduct item reduction.

**Comment by kylefox1** [comment]
@vincenzospaghetti Have you used Confirmatory Factor Analysis before?

**Comment by kylefox1** [comment]
Is anyone proficient in the R programming language?
